CRM/Core/BAO/Setting.php

```diff
@@ -199,6 +199,10 @@ public static function setItems(&$params, $domains = NULL) {
     $fieldsToSet = self::validateSettingsInput($params, $fields);
 
     foreach ($fieldsToSet as $settingField => &$settingValue) {
+      if (empty($fields['values'][$settingField])) {
+        Civi::log()->warning('Deprcated Path There is a setting (' . $settingField . ') not correctly defined. You may see unpredictabilitiy due to this, CRM_Core_Setting::setItems', array('civi.tag' => 'deprecated'));
```

Comment:
I think grepping for them is probably better than trying to maintain a list. Most IDEs will pull up lists from TODO comments in the code, so we could add a todo to help with that